File: app/src/main/java/jp/sourceforge/gokigen/memoma/operations/IObjectSelectionReceiver.java
package jp.sourceforge.gokigen.memoma.operations;


/**
 *   オブジェクトが選択されたことを通知する
 * 
 * @author MRSa
 *
 */
public interface IObjectSelectionReceiver
{
	int touchedVacantArea();
	int touchUppedVacantArea();
	void objectCreated();
    boolean objectSelected(Integer key);
    void objectSelectedContext(Integer key);
}

File: app/src/main/java/jp/sourceforge/gokigen/memoma/operations/IObjectSelectionReceiver.kt
package jp.sourceforge.gokigen.memoma.operations

/**
 * オブジェクトが選択されたことを通知するインタフェース
 */
interface IObjectSelectionReceiver
{
    fun touchedVacantArea(): Int
    fun touchUppedVacantArea(): Int
    fun objectCreated()
    fun objectSelected(key: Int?): Boolean
    fun objectSelectedContext(key: Int?)
}